src/mame/audio/nsub.h

```diff
@@ -0,0 +1,4 @@
+// license:BSD-3-Clause
+// copyright-holders:Zsolt Vasvari, MASH
+
+MACHINE_CONFIG_EXTERN( nsub_audio );
```

Comment:
Please convert this to a device rather than a fragment and a state method.  Look at e.g. one of the Atari sound devices or the Zaccaria sound devices for an example of how to do it.  Having a single method of the driver state class in src/mame/audio doesn't help encapsulation and makes dependencies messier.  In situations like this, if audio/x.cpp to depend on includes/x.h you're doing it wrong.